### ISSUE TITLE
Matrix rate limit reached

### ISSUE THREAD

**Issue by ConorMacBride** [Cadair/matrix-notify-action#1]
A rate limit is being reached, I assume, due to adding lots of reactions in a loop. Maybe a wait could be added between reactions, or handle the error, waiting the suggested time?
```
MatrixHttpClient (REQ-12) {
  errcode: 'M_LIMIT_EXCEEDED',
  error: 'Too Many Requests',
  retry_after_ms: 4087
}
```
https://github.com/sunpy/sunpy/runs/7424617490?check_suite_focus=true
https://github.com/sunpy/sunpy/runs/7424558374?check_suite_focus=true

**Comment by Cadair** [comment]
I am going to keep this open in the hope of a not-hacky fix one day.

**Comment by Cadair** [comment]
I fixed this with a retry in #4
